### PR TITLE
Addition of new attributes to the BD resource

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: mso/resource_mso_schema_template_bd.go [hunk 1]
```diff
@@ -80,6 +80,49 @@ func resourceMSOTemplateBD() *schema.Resource {
...
+				}, false),
+			},
+			"multi_destination_flooding": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"flood_in_bd",
+					"drop",
+					"encap-flood",
```
Comment (lhercot) [inline mso/resource_mso_schema_template_bd.go:99]: encap_flood

File: mso/resource_mso_schema_template_bd.go [hunk 2]
```diff
@@ -95,6 +138,7 @@ func resourceMSOTemplateBD() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+
```
Comment (lhercot) [inline mso/resource_mso_schema_template_bd.go:141]: Why adding an empty line?

File: mso/resource_mso_schema_template_bd.go [hunk 3]
```diff
@@ -394,10 +485,9 @@ func resourceMSOTemplateBDRead(d *schema.ResourceData, m interface{}) error {
 								dhcpPolMap["dhcp_option_policy_version"] = nil
 							}
 						}
-						d.Set("dhcp_policy", dhcpPolMap)
-					} else {
-						d.Set("dhcp_policy", make(map[string]interface{}))
 					}
+					d.Set("dhcp_policy", dhcpPolMap)
+
```
Comment (lhercot) [inline mso/resource_mso_schema_template_bd.go:490]: Remove empty line

File: website/docs/r/schema_template_bd.html.markdown [hunk 4]
```diff
@@ -37,6 +37,12 @@ resource "mso_schema_template_bd" "bridge_domain" {
 * `optimize_wan_bandwidth` - (Optional) Boolean flag to enable or disable the wan bandwidth optimization. Default to false.
 * `layer2_stretch` - (Optional) Boolean flag to enable or disable the layer-2 stretch. Default to false. Should enable this flag if you want to create subnets under this Bridge Domain.
 * `layer3_multicast` - (Optional) Boolean flag to enable or disable layer 3 multicast traffic. Default to false.
+* `arp_flooding` - (Optional) ARP Flooding. Default to false.
+* `virtual_mac_address` - (Optional) Virtual MAC Address.
+* `unicast_routing` - (Optional) Unicast Routing. Default to false.
```
Comment (lhercot) [inline website/docs/r/schema_template_bd.html.markdown:42]: Unicast Routing status

File: website/docs/r/schema_template_bd.html.markdown [hunk 5]
```diff
@@ -37,6 +37,12 @@ resource "mso_schema_template_bd" "bridge_domain" {
 * `optimize_wan_bandwidth` - (Optional) Boolean flag to enable or disable the wan bandwidth optimization. Default to false.
 * `layer2_stretch` - (Optional) Boolean flag to enable or disable the layer-2 stretch. Default to false. Should enable this flag if you want to create subnets under this Bridge Domain.
 * `layer3_multicast` - (Optional) Boolean flag to enable or disable layer 3 multicast traffic. Default to false.
+* `arp_flooding` - (Optional) ARP Flooding. Default to false.
```
Comment (lhercot) [inline website/docs/r/schema_template_bd.html.markdown:40]: ARP Flooding status

File: website/docs/r/schema_template_bd.html.markdown [hunk 6]
```diff
@@ -37,6 +37,12 @@ resource "mso_schema_template_bd" "bridge_domain" {
...
+* `virtual_mac_address` - (Optional) Virtual MAC Address.
+* `unicast_routing` - (Optional) Unicast Routing. Default to false.
+* `ipv6_unknown_multicast_flooding` - (Optional) IPv6 Unknown Multicast Flooding. Allowed values are `flood` and `optimized_flooding`. Default to `flood`.
+* `multi_destination_flooding` - (Optional) Multi-Destination. Allowed values are `flood_in_bd`, `drop` and `encap-flood`. Default to `flood_in_bd`.
+* `unknown_multicast_flooding` - (Optional) Unknown Multicast Flooding. Allowed values are `flood` and `optimized_flooding`. Default to `flood`.
```
Comment (lhercot) [inline website/docs/r/schema_template_bd.html.markdown:45]: Unknown Multicast Flooding behavior.

**Review by lhercot — CHANGES_REQUESTED** [review]

File: mso/resource_mso_schema_template_bd.go (hunk 1 above)
Comment (shrsr) [reply mso/resource_mso_schema_template_bd.go]: changed to flood_in_encap

**Pull request merged**